Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11838,6 +11838,10 @@ void WebPageProxy::immediateActionDidComplete()
 
 void WebPageProxy::didPerformImmediateActionHitTest(const WebHitTestResultData& result, bool contentPreventsDefault, const UserData& userData)
 {
+    if (result.remoteUserInputEventData) {
+        // rdar://127120734 Translate coordinates
+        performImmediateActionHitTestAtLocation(result.remoteUserInputEventData->targetFrameID, result.remoteUserInputEventData->transformedPoint);
```

Comment:
Shouldn’t we return here? Otherwise `didPerformImmediateActionHitTest()` might be called more times than expected.